composer.json

```diff
@@ -27,7 +27,8 @@
     "require": {
         "symfony/cache": "^5.0",
         "guzzlehttp/guzzle": "^6.5",
-        "symfony/dotenv": "^5.0"
+        "symfony/dotenv": "^5.0|^4.0",
+        "linkorb/envoi": "^1.0"
```

Comment:
https://criterion.linkorb.com/linkorb/php-standards/items/165

Reply:
@joostfaassen 
I move packages into  "require-dev" as per my understanding.

Reply:
@kaushikindianic these packages are now in `require`. `require-dev` is empty